### PR TITLE
Rocketchat handler

### PR DESCRIPTION
**Pull request by koallann**
https://github.com/nyaruka/rapidpro/pull/3040

### CODE REVIEW

**Comment by rowanseymour** [comment]
@koallann just to clarify.. this is entirely separate to using RocketChat as a ticketing system right?

**Comment by koallann** [comment]
Yes @rowanseymour

File: handlers/rocketchat/rocketchat.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	if err != nil {
+		return nil, handlers.WriteAndLogRequestError(ctx, h, channel, w, r, err)
+	}
+
+	// check content empty
+	if payload.Text == "" && len(payload.Attachments) == 0 {
+		return nil, handlers.WriteAndLogRequestError(ctx, h, channel, w, r, errors.New("no text or attachment"))
+	}
+
+	urn, err := urns.NewURNFromParts(urns.RocketChatScheme, payload.User.URN, "", payload.User.Username)
```
Comment (rowanseymour) [inline handlers/rocketchat/rocketchat.go:76]: @koallann latest gocommon is now in master if you want to update this PR

Comment (rowanseymour) [reply handlers/rocketchat/rocketchat.go:78]: I meant courier master

Comment (koallann) [reply handlers/rocketchat/rocketchat.go:78]: Sorry, conflicts fixed 👍

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**